app/controllers/users_controller.rb

```diff
@@ -1,4 +1,30 @@
 class UsersController < ApplicationController
+
+  def show
+    @user = User.find params[:id]
```

Comment:
Sự khác nhau giữa find_by and find nói đơn giản thì find_by thì có thể tìm đc kết quả mà không chỉ dùng user id , còn find thì chắc chắn chỉ lấy đc khi biết id . Hơn nữa khi check not found user thi find sẽ trả về recorde not found ( kiểu một lỗi ) còn find_by sẽ trả về cái kết quả nill nhưng trang không bị lỗi , (cách hiểu của em khi đọc xông cái phần bài thôi không biết có đùng không đó:) )